coriolis/tests/api/v1/test_replica_tasks_executions.py

```diff
@@ -0,0 +1,225 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+from webob import exc
+
+from coriolis.api.v1 import replica_tasks_executions as replica_api
+from coriolis.api.v1.views import replica_tasks_execution_view
+from coriolis import exception
+from coriolis.replica_tasks_executions import api
+from coriolis.tests import test_base
+
+
+class ReplicaTasksExecutionControllerTestCase(test_base.CoriolisBaseTestCase):
+    """Test suite for the Coriolis Replica Tasks Execution v1 API"""
+
+    def setUp(self):
+        super(ReplicaTasksExecutionControllerTestCase, self).setUp()
+        self.replica_api = replica_api.ReplicaTasksExecutionController()
+
+    @mock.patch.object(replica_tasks_execution_view, 'single')
+    @mock.patch.object(api.API, 'get_execution')
+    def test_show(
+        self,
+        mock_get_execution,
+        mock_single
+    ):
+        mock_req = mock.Mock()
+        mock_context = mock.Mock()
+        mock_req.environ = {'coriolis.context': mock_context}
+        replica_id = mock.sentinel.replica_id
+        id = mock.sentinel.id
+
+        result = self.replica_api.show(mock_req, replica_id, id)
+
+        self.assertEqual(
+            mock_single.return_value,
+            result
+        )
+
+        mock_context.can.assert_called_once_with(
+            "migration:replica_executions:show")
+        mock_get_execution.assert_called_once_with(
+            mock_context, replica_id, id)
+        mock_single.assert_called_once_with(mock_get_execution.return_value)
+
+    @mock.patch.object(replica_tasks_execution_view, 'single')
+    @mock.patch.object(api.API, 'get_execution')
+    def test_show_not_found(
+        self,
+        mock_get_execution,
+        mock_single
+    ):
+        mock_req = mock.Mock()
+        mock_context = mock.Mock()
+        mock_req.environ = {'coriolis.context': mock_context}
+        replica_id = mock.sentinel.replica_id
+        id = mock.sentinel.id
+        mock_get_execution.return_value = None
+
+        self.assertRaises(
+            exc.HTTPNotFound,
+            self.replica_api.show,
+            mock_req,
+            replica_id,
+            id
+        )
+
+        mock_context.can.assert_called_once_with(
+            "migration:replica_executions:show")
+        mock_get_execution.assert_called_once_with(
+            mock_context, replica_id, id)
```

Comment:
since you already mocked `single`, please also use it:
```suggestion
            mock_context, replica_id, id)
        mock_single.assert_not_called()
```